docs/cheaha/tutorial/index.md

```diff
@@ -0,0 +1,9 @@
+# Getting Started with Using Anaconda on Cheaha
+
+Python is a high level programming language that is widely used in many branches of science. As a result, many scientific packages have been developed in Python, leading to the development of a package manager called Anaconda. Anaconda is the standard in Python package management for scientific research. Consequently Anaconda is used on Cheaha for managing environments and using packages.
```

Comment:
Let's say "Anaconda is a widely used Python package manager..." rather than "Anaconda is the standard". You might be surprised to learn that some folks loathe using Anaconda and prefer pip.